README.md

```diff
@@ -1,2 +1,99 @@
 # PlotEat
-This is an apprenticeship project
+
+PlotEat is an meal planing shopping list app, available at https://ploteat.herokuapp.com/ . 
+
+## Architecture
+
+This is a Rails applicaction that allows you to store recipes, create a menu with your recipes and create a shopping list from the recipes that are on the menu.
+
+## Development
+
+Instructions for how to set up, work with and deploy the project.
+
+
+### Dependencies
+
+This is a
+*[Ruby](https://www.ruby-lang.org)*
+project. `bin/setup` will guide you through installing the required language(s). If you prefer to do this manually, or
+know that you already have the requirements installed `bin/setup` will allow you to skip the language setup and allow
+you to proceed with installing your language(s) manually as you see fit.
+
+All application dependecies can be installed via `bin/setup`. You will have the option to skip each of the below categories.
+To manually install dependencies you can run *`brew bundle`, `bundle install`, `yarn install`, and/or `mix deps.get`*.
+If you prefer another way to manage your dependecies, read the content of the below package files and install them as
+you see fit.
+
+* Native dependencies: [Brewfile](Brewfile)
+* Ruby gems: [Gemfile](Gemfile)
+
+
+***NOTE TO AUTHOR: Rember to remove mentions to package managers not being used in the project, add any special install
+instructions that can't fit in `bin/setup`, and of course edit `bin/setup` to work with your project.***
+
+### Services
+
+- E.g. [Heroku](https://www.heroku.com): Used for hosting
+- E.g. [Code Climate](https://codeclimate.com): Used for code quality
+- E.g. [Travis CI](https://travis-ci.com): Used for running tests before deploys
+
+
+### Secrets
+
+Copy `.env.example` to `.env.development` and ask someone for the correct development secrets.
+
+### Setup
+
+```sh
+$ bin/setup
+```
+
+Seed test data:
+
+```sh
+$ bin/rails db:seed
+```
+
+### Running the project
+
+```sh
+$ bin/rails server
+```
+
+### Tests
+
+Run all tests:
+
+```sh
+$ bin/rake
+```
+Run rspec:
+
+```sh
+$ bin/rspec
+```
+
+Run a single rspec file:
+
+```sh
+$ bin/rspec spec/.../spec.rb
+```
+Run acceptance tests:
+
+```sh
+$ bin/cucumber
```

Comment:
Have you tried running this? :) It doesn't work, since there's no bin stub for cucumber (no file named cucumber inside the folder "bin"). Unless there's a command over at the cucumber readme for generating a bin stub, change this instruction so that is read `bundle exec cucumber`